### DISCUSSION TITLE
Leaver's Edition

### DISCUSSION THREAD

**Discussion by blex-max** [blex-max/ecto_pipe#15] [Announcements]
Our first release - no more git pull! Hopefully at least somewhat stable, at least on Linux. Download according to your OS, unzip and follow instructions as before.

<hr /><em>This discussion was created from the release <a href='https://github.com/kew-myco/its_glue/releases/tag/v1.0.0'>Leaver's Edition</a>.</em>

**Comment by Clarkson1976** [comment]
Thanks for the update Alex

**Comment by catnipping** [comment]
Milestone!

**Comment by blex-max** [comment]
Thanks both! I should probably point out that I mean no more git pull for new installations - otherwise git pull to get the new updates (which is everyone I think)
